lib/engine/game/g_1880/game.rb

```diff
@@ -146,7 +150,9 @@ class Game < Game::Base
                     price: 180,
                     rusts_on: '6',
                     num: 5,
-                    events: [{ 'type' => 'float_30' }, { 'type' => 'permit_b' }],
+                    events: [{ 'type' => 'float_30' },
```

Comment:
Did you remove the permit_b event purposely or merge error?

Reply:
merge error